CHANGELOG.md

```diff
@@ -7,7 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-## [Unreleased](https://github.com/3mdeb/meta-rte/compare/v0.7.3...HEAD)
+## [Unreleased](https://github.com/3mdeb/meta-rte/compare/v0.7.4-rc1...HEAD)
```

Comment:
@PLangowski please, remove this, we will add the Changelog before the release of v0.7.4

Reply:
Same question, is it better if I drop the commit(s) that changed this or should I create new commits that will delete this?